File: Week 01/id_237/rotate-array.java
//给定一个数组，将数组中的元素向右移动 k 个位置，其中 k 是非负数。
class Solution {
    public void rotate(int[] nums, int k) {
        int last = 0;
        int i = 0;
        while(k>0){
            last = nums[nums.length-1];
            for(i = nums.length-1; i >0; --i){
                nums[i] = nums[i-1];
            }
            nums[0] = last;
            --k;
        }
    }
}

File: Week 01/id_262/rotate-array.py
from typing import List


class Solution:
    def rotate(self, nums: List[int], k: int) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        if not nums:
            return None
        for i in range(k):
            nums.insert(0, nums[-1])
            nums.pop(-1)